### ISSUE TITLE
Remove console.log

### ISSUE THREAD

**Issue by cesarandreu** [KyleAMathews/compass-vertical-rhythm#1]
It looks like the v1.3.0 release on npm includes an accidental console.log.

**compass-vertical-rhythm/dist/index.js** (line 107)

``` js
console.log('baseLineheight unit', unit(options.baseLineHeight));
```

**Comment by KyleAMathews** [comment]
Oops! I removed the console.log and then committed but didn't rebuild. Thanks for the notice! Just published 1.3.1

**Comment by cesarandreu** [comment]
Thanks!
